mno_analysis_tools/window_of_downtime.py

```diff
@@ -0,0 +1,70 @@
+# Compute maximum window of time with 0 messages
+import json
+import pytz
+import iso8601
+from datetime import datetime
+from temba_client.v2 import Message
+from core_data_modules.logging import Logger
+
+
+def group(lst, n):
+    """group([0,3,4,10,2,3], 2) => [(0,3), (4,10), (2,3)]
+    
+    Group a list into consecutive n-tuples. Incomplete tuples are
+    discarded e.g.
```

Comment:
Is that the behaviour you want here? It might make more sense to fail if you have incomplete tuples

Reply:
I'll just slice the list into a list of sub-lists